tests/test_data/pred_multi_ens.csv

```diff
@@ -0,0 +1,26 @@
+id,pred_val
+inst_0,"[0.15465898942807543, 0.1139451252954574, 0.19112556436104344, 0.0010999330303991645, 0.5391703878850246]"

```

Comment:
@mkulariya1 can we can generate all the test data related files during tests? During the test runtime, create a tmpdir to save the ground truths and prediction files. Tmpdir will be deleted once the test context becomes out of scope.

Reply:
Now you can delete all these data files

Reply:
done